docs/webperf.md

```diff
@@ -0,0 +1,49 @@
+# Web performance
+
+* TOC
+  {:toc}
```

Comment:
![Zrzut ekranu 2022-03-14 o 10 09 42](https://user-images.githubusercontent.com/22297591/158140434-f6c172bd-3026-44c0-9258-b50e3514cfa3.png)

Is this the desired result?

Reply:
It will be populated during docs deployment to https://allegro.github.io/turnilo/

Reply:
Thanks for the explanation, now we know who doesn't read the documentation 😄